libkbfs/interfaces.go

```diff
@@ -625,7 +625,7 @@ type cryptoPure interface {
 	MakeRandomBranchID() (BranchID, error)
 
 	// MakeMdID computes the MD ID of a RootMetadata object.
-	MakeMdID(md *RootMetadata) (MdID, error)
+	MakeMdID(md *BareRootMetadata) (MdID, error)
```

Comment:
It has to be `*BareRootMetadata*`, since the server side (including `mdserver_memory` and `mdserver_disk` should only work with `*BareRootMetadata`, and they calculate MdIDs too.